tests/robot-tests/tests/visual_testing/tables_and_charts.py

```diff
@@ -0,0 +1,166 @@
+import csv
+import os
+from enum import Enum
+from logging import warning
+
+releases_by_url = {}
```

Comment:
Is this just a dictionary of release URLs?

Reply:
yeah, dictionary with release urls as the key and Releases as the value